### PR TITLE
Add GPT-3 175B v5p MLPerf 4.0 scripts

### PR DESCRIPTION
**Pull request by anfals**
Adding this for easier reproducibility by customer engineers

### CODE REVIEW

File: MaxText/configs/v5p/gpt3_175b/gpt3_175b_base.sh [hunk 1]
```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+# GPT-3 175B Model.
+# This script abstracts away all the shared config between Google's various MLPerf 4.0 submissions, which 
+# achieved high performance on the Training benchmark. See the various different TPU type configs in this 
+# directory to actually run the training.
+
+set -euox pipefail
+
+bash preflight.sh PLATFORM=gke
+sleep 60
```
Comment (gobbleturk) [inline MaxText/configs/v5p/gpt3_175b/gpt3_175b_base.sh:10]: Why is this sleep needed? We don't run sleep in other configs?

File: MaxText/configs/v5p/gpt3_175b/gpt3_175b_base.sh [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+# Read arguments or use defaults from environment variables
+PER_DEVICE_BATCH_SIZE=${1:-${PER_DEVICE_BATCH_SIZE:-4}}
+REMAT_POLICY=${2:-${REMAT_POLICY:-full}}
+ICI_DATA_PARALLELISM=${3:-${ICI_DATA_PARALLELISM:-1}}
+ICI_FSDP_PARALLELISM=${4:-${ICI_FSDP_PARALLELISM:-64}}
+ICI_TENSOR_PARALLELISM=${5:-${ICI_TENSOR_PARALLELISM:-8}}
+RUNNAME=${6:-${RUNNAME:-convergence_test_0}}
+BASE_OUTPUT_DIRECTORY=${7:-${BASE_OUTPUT_DIRECTORY:-gs://some-bucket}}
+
+python3 MaxText/train.py MaxText/configs/base.yml run_name="${RUNNAME}" model_name=gpt3-175b\
```
Comment (gobbleturk) [inline MaxText/configs/v5p/gpt3_175b/gpt3_175b_base.sh:33]: Can we allow either train or train_compile here like in our other config files, e.g. https://github.com/google/maxtext/blob/a2553fa809592a55251fc6f89ef033c264eda72f/MaxText/configs/v5p/128b.sh#L44

**Review by gobbleturk — APPROVED** [review]
Awesome, thank you for adding these!

File: MaxText/configs/v5p/gpt3_175b/gpt3_175b_base.sh (hunk 1 above)
Comment (anfals) [reply MaxText/configs/v5p/gpt3_175b/gpt3_175b_base.sh:10]: I based these on the MLPerf scripts we have internally and they had this. I can test without it Monday and see if it works and then update

File: MaxText/configs/v5p/gpt3_175b/gpt3_175b_base.sh (hunk 2 above)
Comment (anfals) [reply MaxText/configs/v5p/gpt3_175b/gpt3_175b_base.sh:33]: Ah that'll be useful! I'd want to test the script with that addition before merging and these scripts need to urgently get in for a Training on Monday, so I'll test that addition on Monday :)

**Pull request merged**